testing/src/main/java/org/conscrypt/TestVector.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.conscrypt;
+
+import static org.conscrypt.TestUtils.decodeHex;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public final class TestVector {
+    private final Map<String, String> map = new HashMap<>();
+
+    public void put(String label, String value) {
+        map.put(label, value);
```

Comment:
Are overrides expected?

Reply:
Sorry, I've meant overrides in the underlying map's values: duplicate labels are probably a bug.

Reply:
Oh right... For some reason I was thinking Map.put() threw on dups, so yeah, we should handle that.  I'll probably go a step further and allow a way to specify the schema... Just not this PR.